configs/server/forseti_conf_server.yaml.in

```diff
@@ -54,6 +54,10 @@ inventory:
           max_calls: 4
           period: 1.2
           disable_polling: False
+        groupssettings:
+          max_calls: 14
```

Comment:
How did you determine this, if there is a max quota of 100K per day?

Reply:
this is ultimately part of admin api and so it should be the same as what we have for that no?